### PR TITLE
OSDOCS-8372: Migrate "Getting Started with Rosa - Deploy the Cluster (Simple CLI)"

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-8372](https://issues.redhat.com//browse/OSDOCS-8372): Migrate "Getting Started with Rosa - Deploy the Cluster (Simple CLI)" from rosaworkshop.io content.

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-8372

Link to docs preview:
https://68105--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @yuwang-RH! Can I please get a review of this PR? Thanks!

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+//Brought into ROSA product docs 2023-11-16
+
+This page outlines the minimum list of commands to deploy a ROSA cluster using the command line interface (CLI). 
+
+[NOTE]
+====
+While this simple deployment works well for a workshop setting, clusters used in production should be deployed with a more detailed method.
+====
+
+== Prerequisites
```
Comment (mletalie) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide.adoc:19]: Should "Prerequisites" be formatted in this way? 
image

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+= Tutorial: Simple CLI guide
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-simple-cli-guide
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-11-16
+
+This page outlines the minimum list of commands to deploy a ROSA cluster using the command line interface (CLI).
```
Comment (mletalie) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide.adoc:12]: Do we need to spell out "Red Hat OpenShift on AWS" at least once on these tutorials?

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide.adoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+Run the following command _once_ for each AWS account and y-stream OpenShift version:
+
+[source,terminal]
+----
+rosa create account-roles --mode auto --yes
+----
+
+== Deploying the cluster
+
+. Create the cluster with the default configuration by running the following command substituting your own cluster name:
```
Comment (mletalie) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide.adoc:33]: Do we need to point out that in the example given, this is specifically an STS cluster? Not sure how much detail there needs to be in these tutorials? 


```suggestion
. Create the cluster with the default configuration by running the following command, substituting your own cluster name:
```

**Comment by mletalie** [comment]
Hello @jneczypor! Just a few suggestions/comments, otherwise, LGTM! 
Good job!

**Review by mletalie — COMMENTED** [review]
/lgtm

**Review by mletalie — COMMENTED** [review]
/lgtm

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide.adoc (hunk 3 above)
Comment (jneczypor) [reply cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide.adoc:33]: Yes, you are totally correct. One of the previous tutorials in this "workshop" says this is an STS-cluster. We assume they will follow the entire workshop, so no need to repeat STS.

**Pull request merged**